Document/MediaManager.php

```diff
@@ -34,7 +34,7 @@ public function save($entity, $andFlush = true)
             $entity->setProviderName(func_get_arg(2));
         }
 
-        if ($andFlush && is_bool($andFlush)) {
+        if (is_bool($andFlush)) {
```

Comment:
Just asking: With this condition, `false` and `true` would pass as `$andFlush` is a boolean.

Would be more logic to have:

```php
is_bool($andFlush) && $andFlush
```

Ore more simple:

```php
true === $andFlush
```

Strict comparison will check both type and value.

What do you think?